db/model/Gdoc/rawToArchie.ts

```diff
@@ -27,7 +27,7 @@ import {
 import { match } from "ts-pattern"
 
 function appendDotEndIfMultiline(line: string): string {
-    if (line.includes("\n")) return line + "\n.end"
+    if (line.includes("\n")) return line + "\n:end"
```

Comment:
I didn't catch the reason for this on first pass, so I'm not sure about this change

Reply:
It was a mistake in the original code. If you look at the archieml spec, it says that simple property values can be multiline but then they have to have the `:end` delimiter. From an example on the spec website: 

```
key: value
 More value

Even more value
:end 
```

Reply:
ah gotcha, thanks for the pointers!